public/app/features/visualization/data-hover/DataHoverView.tsx

```diff
@@ -63,7 +62,8 @@ export function getDisplayValuesAndLinks(
   const linkLookup = new Set<string>();
 
   for (const field of orderedVisibleFields) {
-    if (mode === TooltipDisplayMode.Single && columnIndex != null && !field.hovered) {
+    const isFieldHovered = columnIndex === fields.indexOf(field);
```

Comment:
ah looking at this we could probably replace indexOf check with just `field === hovered` -> though maybe we should rename `hovered` to `hoveredField` or something to improve readability 😬